src/dmd/frontend.h

```diff
@@ -6850,12 +6872,7 @@ struct Param
     DArray< const char > moduleDepsFile;
     OutBuffer* moduleDeps;
     MessageStyle messageStyle;
-    bool debugb;
-    bool debugc;
-    bool debugf;
-    bool debugr;
-    bool debugx;
-    bool debugy;
+    DMDparams dmdParams;
```

Comment:
Well having it in global defies the point of it not being in common frontend options. If importing mars is problematic, then define it in a new module.